cmd/storj-sim/network.go

```diff
@@ -205,6 +211,21 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			}
 
 			// TODO: maybe all the config flags should be exposed for all processes?
+			apiKey := vip.GetString("client.api-key")
+			if apiKey == "" {
+				parts := strings.Split(satellite.Address, ":")
+				consoleAddress := fmt.Sprintf("http://%s:%d/api/graphql/v0", parts[0], consolePort)
```

Comment:
there can be multiple satellites and consoles running concurrently on the same machine

Reply:
also users can later change both the satellite and console address in the config